new_add_new_naming_numbered_parameter_assignment_cop.md

```diff
@@ -0,0 +1 @@
+* [#9999](https://github.com/rubocop-hq/rubocop/pull/9999): Add new `Lint/NumberedParameterAssignment` cop. ([@koic][])
```

Comment:
I'm not sure what this file is. It shouldn't be here, right?

Reply:
Oops! This is a mistake and I removed this file 💦